doc/compiler/jitserver/Problem.md

```diff
@@ -19,9 +19,40 @@ OpenJDK Assembly Exception [2].
 
 SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 -->
+# Debugging JITServer
 
-Usually typical OpenJ9 debugging procedures can be followed for JITServer. But there are a few JITServer specific tips and tricks.
+Typical OpenJ9 debugging procedures can be followed for JITServer. But there are a few JITServer specific tips and tricks.
+Unlike regular OpenJ9, where everything runs in one process, JITServer separates JIT compiler into a separate process while constantly
```

Comment:
"Unlike regular OpenJ9 where everything runs in one process, JITServer technology moves the JIT compiler activity into a separate process while ..."